frontend/src/hooks/useCardSearch.js

```diff
@@ -0,0 +1,22 @@
+import {useState} from "react";
+import axios from "axios";
+
+export default function useCardSearch(){
+
+    const [cards, setCards] = useState([]);
+
+    const getSpecificCard = (cardName) => {
+        return axios
+            .get('/api/cards?cardName=' + cardName)
+            .then(response => response.data)
+            .then ((data) => setCards(data))
```

Comment:
```suggestion
            .then (setCards)
```

Reply:
thanks, I will consider this for the next pr